actions/class.RestResourceComments.php

```
@@ -0,0 +1,266 @@
+
+/**
+ * Authoring comments REST API (FR1) — Item, Test, or Asset via resourceType.
+ *
+ * Routes:
+ * - GET  /taoItems/RestResourceComments/index?resourceUri=&resourceType=
+ * - POST /taoItems/RestResourceComments/index  (resourceUri, resourceType, body)
+ * - POST /taoItems/RestResourceComments/update (id, body) — author can edit own comment
+ * - POST /taoItems/RestResourceComments/resolve (id, resolved) — any authenticated authoring user
+ * - POST /taoItems/RestResourceComments/delete (id) — author can delete own comment
+ */
+class taoItems_actions_RestResourceComments extends tao_actions_CommonModule
```

PHPStan (2.2.7)

[error] Class taoItems_actions_RestResourceComments extends unknown class tao_actions_CommonModule.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)